README-QUIC.md

```diff
@@ -25,43 +26,15 @@ FAQ
 ### Why would I want to use QUIC, and what functionality does QUIC offer relative to TLS or DTLS?
 
 QUIC is a state-of-the-art secure transport protocol carried over UDP. It can
-serve many of the use cases of TLS as well as those of DTLS. QUIC delivers
-a number of advantages:
-
-- It supports multiple streams of communication, allowing application protocols
-  built on QUIC to create arbitrarily many bytestreams for communication between
-  a client and server. This allows an application protocol to avoid head-of-line
-  blocking and allows an application to open additional logical streams without
-  any round trip penalty, unlike opening an additional TCP connection.
-
-- Since QUIC is the basis of HTTP/3, support for QUIC also enables applications
-  to use HTTP/3 using a suitable third-party library.
-
-- Future versions of OpenSSL will offer support for 0-RTT connection
-  initiation, allowing a connection to be initiated to a server and application
-  data to be transmitted without any waiting time. This is similar to TLS 1.3's
-  0-RTT functionality but also avoids the round trip needed to open a TCP
-  socket; thus, it is similar to a combination of TLS 1.3 0-RTT and TCP Fast
-  Open.
-
-- Future versions of OpenSSL will offer support for connection
-  migration, allowing connections to seamlessly survive IP address changes.
-
-- Future versions of OpenSSL will offer support for the QUIC
-  datagram extension, allowing support for both TLS and DTLS-style use cases on
-  a single connection.
-
-- Because most QUIC implementations, including OpenSSL's implementation, are
-  implemented as an application library rather than by an operating system, an
-  application can gain the benefit of QUIC without needing to wait for an OS
-  update to be deployed. Future evolutions and enhancements to the QUIC protocol
-  can be delivered as quickly as an application can be updated without
-  dependency on an OS update cadence.
-
-- Because QUIC is UDP-based, it is possible to multiplex a QUIC connection
-  on the same UDP socket as some other UDP-based protocols, such as RTP.
-
-For more background information on OpenSSL's QUIC implementation, see the
+serve many of the use cases of SSL/TLS as well as those of DTLS.
+
+QUIC delivers a number of advantages such as support for multiple streams of
+communication; it is the basis for HTTP/3 [RFC 9114]; fast connection
+initiation; and connection migration (enabling a connection to survive IP
+address changes). For a more complete description of what QUIC is and its
+advantages see the [QUIC Introduction] in the [OpenSSL Guide].
+
+For a more comprehensive overview of OpenSSL's QUIC implementation, see the
```

Comment:
I think this sentence reads better if it does not start in the same way as the previous sentence.

maybe reword and drop "more":  "For a comprehensive overview...."